plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSourceProvider.java

```diff
@@ -249,7 +248,7 @@ public ConnectorPageSource createPageSource(
                 hiveColumnHandles.build(),
                 ImmutableList.of(parquetPredicate),
                 true,
-                parquetDateTimeZone,
+                UTC,
```

Comment:
*suggestion (non-blocking)*: this does not seem to be related to Hive - can/should this go in a separate commit? Also looks like we now are hardcoding some default - could you please add some context (in a new commit message)?

Reply:
ya why this change?

Reply:
Explanation https://github.com/trinodb/trino/pull/22577#discussion_r1680404943